src/common/src/common-b64.c

```diff
@@ -262,14 +262,20 @@ static const uint8_t mongoc_b64rmap_invalid = 0xff;
 #if defined(BSON_OS_UNIX)
```

Comment:
I expect this entire `#if / #else / #endif` block can be removed. Add `#include <common-thread-private.h>` and instead use the `bson_once*` and `BSON_ONCE*` APIs. `bson_once` already has an assert.